src/dotnet/commands/dotnet-cache/LocalizableStrings.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Microsoft.DotNet.Tools.Cache
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Cache";
+
+        public const string AppDescription = "Cahes Assemblies specified for the .NET Platform, By Default these will be optimized for the target runtime and framework";
+
+        public const string ProjectEntries = "ProjectEntries";
+
+        public const string ProjectEntryDescription = "The MSBuild project file used to specify the list of packages to be cached.";
+
+        public const string FrameworkOption = "FRAMEWORK";
+
+        public const string FrameworkOptionDescription = "Target framework for which to cache for";
+
+        public const string RuntimeOption = "RUNTIME_IDENTIFIER";
+
+        public const string RuntimeOptionDescription = "Target runtime to cache for.";
+
+        public const string OutputOption = "OUTPUT_DIR";
+
+        public const string OutputOptionDescription = "Path in which to cache the given assemblies";
+    
+        public const string FrameworkVersionOption = "FrameworkVersion";
+
+        public const string FrameworkVersionOptionDescription = "The Microsoft.NETCore.App package version that will be used to run the assemblies";
+
+        public const string SkipOptmizationOptionDescription = "This is will skip the optimization phase";
```

Comment:
const name has a typo. It should be SkipOptimizationOptionDescription not Optmization.

Suggested rewriting: "Skips the optimization phase."